README.md

```diff
@@ -91,3 +93,20 @@ delete-scxa-gene2experiment-exp-entries.sh
 ## Tests
 
 Tests are located in the `tests` directory and use bats. To run them, execute `bash tests/run-tests.sh`. The `tests` folder includes example data in matrix markt format.
+
+# Container
+
+The container is available for use at quay.io/ebigxa/index-scxa-module at latest or any of the tags after 0.2.0, so it could be used like this for example:
+
+```
+docker run -v /local_data:/data \
+       -e dbConfig=<your-database-connection-string-for-postgres> \
+       -e EXP_ID=<the-accession-of-experiment> \
+       -e SOLR_HOST=<solr-host:solr-port> \
+       -e MATRIX_MARKT_ROWS_GENES_FILE=<path-inside-container-for-matrixMarkt-file> \
+       --entrypoint load_scxa_gene2experiment_index.sh \
+       quay.io/ebigxa/index-scxa-module:latest
+```
```

Comment:
Why do we need the database config string? Is this used elsewhere?

Reply:
hehehe... this was copy/past drag error, thanks for spotting it!